src/qt/askpassphrasedialog.cpp

```diff
@@ -104,10 +104,15 @@ void AskPassphraseDialog::accept()
             // Cannot encrypt with empty passphrase
             break;
         }
-        QMessageBox::StandardButton retval = QMessageBox::question(this, tr("Confirm wallet encryption"),
-                 tr("Warning: If you encrypt your wallet and lose your passphrase, you will <b>LOSE ALL OF YOUR BITCOINS</b>!") + "<br><br>" + tr("Are you sure you wish to encrypt your wallet?"),
-                 QMessageBox::Yes|QMessageBox::Cancel,
-                 QMessageBox::Cancel);
+        QMessageBox msgBoxConfirm(QMessageBox::Question,
+                                  tr("Confirm wallet encryption"),
+                                  tr("Warning: If you encrypt your wallet and lose your passphrase, you will <b>LOSE ALL OF YOUR BITCOINS</b>!<br><br>"
+                                     "Are you sure you wish to encrypt your wallet?"),
```

Comment:
Could we avoid changes in translatable strings in this PR?

Reply:
Yes, will do.

Reply:
I changed the code to reuse already existing strings, can you confirm it's okey in this new way ?